full-stack-tests/presentation/src/frontend/Content.test.ts

```diff
@@ -399,6 +399,44 @@ describe("Content", () => {
 
   });
 
+  describe("Guid properties", () => {
+
+    it("creates guid fields", async function () {
+      const guid = Guid.createValue();
+      let instanceKey: InstanceKey;
+      const imodelConnection = await buildTestIModelConnection(this.test!.fullTitle(), (db) => {
+        instanceKey = insertDocumentPartition(db, "Test", undefined, guid);
+      });
+
+      const ruleset: Ruleset = {
+        id: Guid.createValue(),
+        rules: [{
+          ruleType: RuleTypes.Content,
+          specifications: [{
+            specType: ContentSpecificationTypes.SelectedNodeInstances,
+            propertyOverrides: [{
+              name: "FederationGuid",
+              isDisplayed: true,
+              doNotHideOtherPropertiesOnDisplayOverride: true
```

Comment:
I believe in this test we only care about the `FederationGuid` property, so don't need others to be loaded too. In that case, the `doNotHideOtherPropertiesOnDisplayOverride` attribute is unnecessary.

Reply:
fixed